Jenkinsfile

```diff
@@ -435,10 +448,13 @@ pipeline {
             values '3.0',     // latest 3.0.x Apache Cassandra�
                   '3.11',    // latest 3.11.x Apache Cassandra�
                   '4.0',    // latest 4.0.x Apache Cassandra�
-                  '5.0-beta1', // Development Apache Cassandra�
+                  '4.1',
+                  '5.0', // Development Apache Cassandra�
                   'dse-5.1.35', // latest 5.1.x DataStax Enterprise
                   'dse-6.7.17', // latest 6.7.x DataStax Enterprise
-                  'dse-6.8.30' // 6.8 current DataStax Enterprise
+                  'dse-6.8.30', // 6.8 current DataStax Enterprise
+                  'dse-6.9.3',
+                  'hcd-1.0.0'
```

Comment:
We need to prune this a bit, the matrix is too long for a "per-commit" build. We can run all the versions we want on the daily/weekly builds.

Maybe we can use `3.11`,  `4.1`, `5.0`, `dse-5.1`, `dse-6.9.3` and `hcd-1.0.0` for `per-commit`